### DISCUSSION TITLE
IOS / Swift / Error on step "Set up provisioning profiles settings on Xcode project"

### DISCUSSION THREAD

**Discussion by bukin26** [codemagic-ci-cd/codemagic-docs#2778] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

Start IOS workflow build from Codemagic

### Expected results

Publishing to AppStore

### Actual results

Publishing to AppStore failed with error:

"Step 3 script `Set up provisioning profiles settings on Xcode project` exited with status code 1"

![Снимок экрана 2024-10-09 в 22 56 07](https://github.com/user-attachments/assets/1e35e8bf-736d-4e38-919a-8e0f4c169534)

Previously I used codemagic.yaml with same structure and everything was fine. But now build has failed.
Also i see that in previous step certificate and profile were saved.

![Снимок экрана 2024-10-09 в 22 59 29](https://github.com/user-attachments/assets/68a116c0-6694-41ae-8782-b3089fd7918f)

### Build id (optional)

6706dccc1a0130f2a6b45e4e

**Comment by bukin26** [comment]
I also found the following information:

Xcode 16 changes the location for the installation of the Provisioning Profiles.
In previous Xcode releases, these were installed under the path: ~/Library/MobileDevice/Provisioning Profiles
Starting with Xcode 16, these are installed under: ~/Library/Developer/Xcode/UserData/Provisioning Profiles

I am using version 16 of Xcode, but from the logs I see that the Profile is saving the old location. Could this be the problem?

**Reply by Nodirbek-Khudoyberdiev** [reply]
have you found a solution ?

**Comment by dam-ease** [comment]
Hello @bukin26,
Can you confirm that you have the file `CoolFishin.xcodeproj` in you project directory? 
Looking through your projects repo on github, I see the `.xcodeproj` file is `CoolFis.xcodeproj` instead and also see you are setting your `XCODE_SCHEME` to be `CoolFishin` and not `CoolFis`, this might be where the issue lies, can you please check this out?

**Reply by dam-ease** [reply, marked as answer]
Hey @bukin26,
Any update for me on this? An update on a similar issue has been shared [here](https://github.com/orgs/codemagic-ci-cd/discussions/2787#discussioncomment-11073328). If there is anything else I can assist you with, please don't hesitate to let me know.
